react-carousel-component/dist/styles.css

```diff
@@ -0,0 +1,53 @@
+* {
+  box-sizing: border-box;
+}
+
+#root {
+  display: flex;
+  margin: 0 auto;
+  align-items: center;
+  height: 100vh;
+}
+
+.fa-chevron-right,
+.fa-chevron-left {
+  font-size: 2.5rem;
+  padding: 1rem;
+  cursor: pointer;
+}
+
+.fa-chevron-right {
+  margin-left: 3rem;
+}
+
+.fa-chevron-left {
+  margin-right: 3rem;
+}
+
+.fa-circle {
+  font-size: 1.5rem;
+  padding: 0.75rem;
+  cursor: pointer;
+}
```

Comment:
## ☠️  Never Modify 3rd-Party CSS Classes ☠️ 
It's not a good idea to style 3rd-party classes. Instead add your own CSS class. Overriding the Font Awesome styling will confuse the hell out of your teammates, since they no longer behave as stated in the Font Awesome documentation.